### PR TITLE
Make hybrid and use test variants

### PR DESCRIPTION
**Pull request by alshakero**
I disabled variants test and only testing under v0. Because the task isn't as simple as it sounds. We haven't tested `juicy-html` nor `imported-template` under V1 polyfill and it doesn't seem trivial. More than half of the tests fail under v1.

We also need to figure a way to detect v0 vs v1. Other than that I think the element is properly hybrid.

### CODE REVIEW

File: bower-polymer2.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  "variants": {
+    "polymer2": {
+      "directory": "bower_components-polymer2",
+      "devDependencies": {
+        "polymer": "^2.0.0"
+      }
+    }
+  },
+  "directory": "bower_components-polymer2"
+}
```
Comment (warpech) [inline bower-polymer2.json:49]: Something went wrong with this file. You committed it to Git and added it to `.gitignore` simultaneously.

File: bower.json [hunk 2]
```diff
@@ -36,5 +35,13 @@
...
     "polymer": "^1.9.1",
     "document-register-element": "^1.5.0"
+  },
+  "variants": {
+    "polymer2": {
+      "directory": "bower_components-polymer2",
+      "devDependencies": {
+        "polymer": "^2.0.0"
+      }
+    }
```
Comment (warpech) [inline bower.json:45]: Why not do the variants identically as Polymer does it? https://github.com/PolymerElements/iron-list/blob/master/bower.json

File: package.json [hunk 3]
```diff
@@ -13,11 +13,12 @@
...
     "grunt-contrib-watch": "^0.6.1",
     "path": "^0.12.7",
-    "web-component-tester": "github:tomalec/web-component-tester#88236713a7"
+    "web-component-tester": "github:tomalec/web-component-tester#88236713a7",
+    "polymer-cli": "^1.5.7"    
   },
   "scripts": {
     "serve": "npm run mockserver && polyserve",
-    "polyserve": "polyserve -o --proxy-target http://127.0.0.1:9999/sc --proxy-path sc",
+    "polyserve": "polyserve -o --proxy-target http://127.0.0.1:9999/sc --proxy-path sc --hostname 0.0.0.0",
```
Comment (warpech) [inline package.json:21]: I'd rather just remove `-o`. Is there any benefit from it?

File: starcounter-include.html [hunk 4]
```diff
@@ -63,26 +64,32 @@
...
-                this.stampImportedTemplate();
+            // computed method name (http://2ality.com/2015/02/es6-classes-final.html#computed-method-names)
+            [WebComponentsV0 ? 'attachedCallback' : 'connectedCallback']() {
+                this.stampImportedTemplate();   
             }
-        }
+            [WebComponentsV0 ? 'createdCallback' : 'noop']() {                
+                this.createComponent();
+                // in V0, attributeChangedCallback isn't called with the initial values, we do that manually
+                Array.prototype.forEach.call(this.attributes, attr => this.attributeChangedCallback(attr.name, null, attr.value));
```
Comment (warpech) [inline starcounter-include.html:88]: Shouldn't this line be skipped for CEv1?

File: starcounter-include.html [hunk 5]
```diff
@@ -14,14 +14,15 @@
 
 <script>
     (function() {
-        const useShadowDOMV1 = Boolean(Element.prototype.attachShadow && Node.prototype.getRootNode);
+        const WebComponentsV0 = true; // need a way to detect WC version
```
Comment (warpech) [inline starcounter-include.html:17]: Why is this a problem? Can't you just check if `window.customElements !== undefined`? 

You can assume the `d-r-e.js` will be removed from Starcounter 2.3.1 (I assume we have no other option as it is not compatible with `wc.js V0`).

**Review by warpech — CHANGES_REQUESTED** [review]
Please also add instructions to `CONTRIBUTING.md` how to  run `bower install` to configure a V0 or V1 environment for development.

File: starcounter-include.html (hunk 4 above)
Comment (alshakero) [reply starcounter-include.html:88]: It is, `noop` is never called.

**Comment by alshakero** [comment]
Closing in favour of https://github.com/Starcounter/starcounter-include/pull/66

**Pull request closed without merging**